From 8e0d407f2bf5008483c769edb96417c06ef00a34 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Thu, 10 Feb 2022 03:07:20 +0100
Subject: [PATCH 377/456] usb: typec: typec-extcon: Port to multi-device
 mux/switch interfaces

Just to make it compile, while I'll be playing with the upstreamable
solution.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/usb/typec/typec-extcon.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/usb/typec/typec-extcon.c b/drivers/usb/typec/typec-extcon.c
index 447aaafca0c9..e460f35c8390 100644
--- a/drivers/usb/typec/typec-extcon.c
+++ b/drivers/usb/typec/typec-extcon.c
@@ -23,8 +23,8 @@ struct typec_extcon {
 
 	/* consumers */
 	struct usb_role_switch *role_sw;
-        struct typec_switch *sw;
-        struct typec_mux *mux;
+        struct typec_switch_dev *sw;
+        struct typec_mux_dev *mux;
 
 	/* providers */
 	struct extcon_dev *extcon;
@@ -134,7 +134,7 @@ static int typec_extcon_sync_extcon(struct typec_extcon *tce)
 	return 0;
 }
 
-static int typec_extcon_sw_set(struct typec_switch *sw,
+static int typec_extcon_sw_set(struct typec_switch_dev *sw,
 			       enum typec_orientation orientation)
 {
         struct typec_extcon *tce = typec_switch_get_drvdata(sw);
@@ -150,7 +150,7 @@ static int typec_extcon_sw_set(struct typec_switch *sw,
         return 0;
 }
 
-static int typec_extcon_mux_set(struct typec_mux *mux,
+static int typec_extcon_mux_set(struct typec_mux_dev *mux,
 				struct typec_mux_state *state)
 {
         struct typec_extcon *tce = typec_mux_get_drvdata(mux);
-- 
2.34.1

